### DISCUSSION TITLE
Maintenance mode?

### DISCUSSION THREAD

**Discussion by AlexBa** [shopware/frontends#547] [General]
Hello everyone,
Is it currently possible to use the maintenance mode per sales channel? And if not, is this feature planned?

Kind regards
Alexander Barton

**Comment by mkucmus** [comment]
hey, you can enable the `maintenance mode` in the Sales Channel Settings:
![image](https://github.com/shopware/frontends/assets/5596960/46c169ed-d03c-4caf-8dea-271b62bc42ba)

then, every request to the `store-api` will get `HTTP 503` as a response with some more details, like:
```
    "errors": [
        {
            "status": "503",
            "code": "FRAMEWORK__API_SALES_CHANNEL_MAINTENANCE_MODE",
            "title": "Service Unavailable",
            "detail": "The sales channel is in maintenance mode.",
```

you can display some more information by adjusting the [error.vue](https://github.com/shopware/frontends/blob/main/templates/vue-demo-store/error.vue) page, for 503 case specifically (see the error code's map inside the file) if you don't like the general solution
![image](https://github.com/shopware/frontends/assets/5596960/ba3f8640-ce0d-49a0-9be3-c5084cff0832)
